.github/workflows/lint-test.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         run: "ruff check --output-format=github ."
 
       - name: Run tests and generate coverage report
-        run: pytest -n auto --cov -q
+        run: COVERAGE_CORE=sysmon pytest -n auto --cov -q
```

Comment:
Looks good, my only thing would be I prefer to use a designated env block, then we can document it.

```suggestion
        run: pytest -n auto --cov -q
        env:
          # Use "sys.monitoring" based coverage backend for better speed (see #3075)
          COVERAGE_CORE: sysmon
```

Reply:
Good idea, committed your suggeston.